From 5f15d510969d14928309511e04b5c012b66ad36d Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Wed, 27 Feb 2019 18:17:32 -0700
Subject: [PATCH 12/12] effects: Initialize volume at -96

* -96 is the correct value to mute the stream, and appears to work
  without any side effects on devices, like a loud burst of audio
  followed by silence (or very little audio) after a pause and resume

Change-Id: I566726828caf3f6a05816db21ca4074800350346
---
 media/libeffects/lvm/wrapper/Bundle/EffectBundle.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/media/libeffects/lvm/wrapper/Bundle/EffectBundle.cpp b/media/libeffects/lvm/wrapper/Bundle/EffectBundle.cpp
index 04c269228..a8152a11a 100644
--- a/media/libeffects/lvm/wrapper/Bundle/EffectBundle.cpp
+++ b/media/libeffects/lvm/wrapper/Bundle/EffectBundle.cpp
@@ -256,7 +256,7 @@ extern "C" int EffectCreate(const effect_uuid_t *uuid,
         pContext->pBundledContext->NumberEffectsEnabled     = 0;
         pContext->pBundledContext->NumberEffectsCalled      = 0;
         pContext->pBundledContext->firstVolume              = LVM_TRUE;
-        pContext->pBundledContext->volume                   = 0;
+        pContext->pBundledContext->volume                   = -96;
 
         #ifdef LVM_PCM
         char fileName[256];
-- 
2.17.1

